website/src/blogs/running-through-the-mud-looking-skywards.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Running through the mud looking skywards
+author: Amit Kumar Das
+author_info: Engineer the DAO
+date: 2017-02-11
+tags: Education, Industry, Storage, Openebs
+excerpt: I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical.
+---
+
+I knew, we would be running through the mud looking skywards for those brightest stars. We were not at all disappointed with our first attempt at *Kuppam Engineering College*, India.
+
+I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical. We call ourselves as the industry’s core and form the industry’s grey cells. What we need to do is make our problems open to the public. These problems becomes the new practice that needs to be picked up by our *alma mater*. Once selected, it will take no time for these problems to go viral among the students of these institutions. Let these young minds chart their own course; love, enjoy & live every moment when they traverse through this meandering amazon of practical problems. This becomes the fodder for the future theory allowing the current one to rest in peace.
```

Comment:
```suggestion
I always believe, bridging the minds of academia & industry has to go beyond the nuances of theory vs. practical. We call ourselves the industry’s core and form the industry’s grey cells. What we need to do is make our problems open to the public. These problems becomes the new practice that needs to be picked up by our *alma mater*. Once selected, it will take no time for these problems to go viral among the students of these institutions. Let these young minds chart their own course; love, enjoy & live every moment when they traverse through this meandering amazon of practical problems. This becomes the fodder for the future theory allowing the current one to rest in peace.
```